apps/frontend/src/extra_components/Carousel.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useMemo, useState } from "react";
+import {
+  Box,
+  IconButton,
+  Card,
+  Slide,
+  Typography,
+  Grid,
+  Paper,
+} from "@mui/material";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+
+export interface props {
+  imageURLs: string[];
+}
+const Carousel = ({ imageURLs }: props) => {
+  const [cards, setCards] = useState<React.ReactElement[]>([]);
+  const [currentPage, setCurrentPage] = useState(0);
+  const [slideDirection, setSlideDirection] = useState<
+    "right" | "left" | undefined
+  >("left");
+
+  const images: React.ReactElement[] = useMemo(() => {
+    return Array.from({ length: imageURLs.length }, (_, i) => <Card key={i} />);
+  }, [imageURLs]);
+
+  const handleNextPage = () => {
+    setSlideDirection("left");
+    setCurrentPage((prevPage) => prevPage + 1);
+  };
+
+  const handlePrevPage = () => {
+    setSlideDirection("left");
+    setCurrentPage((prevPage) => prevPage - 1);
+  };
+
+  useEffect(() => {
+    setCards(images);
+  }, [images]);
+
+  return (
+    <Paper
+      sx={{
+        display: "flex",
+        flexDirection: "row",
+        alignItems: "center",
+        padding: "5%",
+        margin: "5%",
+        width: "85%",
+      }}
+    >
+      <Grid
+        container
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "center",
+          alignContent: "center",
+          justifyContent: "center",
+          width: "100%",
+          height: "50vh",
```

Comment:
not a huge fan of using vh here, but it's not necessary to change as long as it displays properly on mobile and desktop

Reply:
Sounds good, I'll switch that to px or %, whichever works better!